src/AvtaleSide/GodkjenningSteg/Oppsummering/instruks/instruks.less

```diff
@@ -1,17 +1,24 @@
+@import '~nav-frontend-core/less/_variabler';
 .instruks {
     &__subheader {
-        padding: 0 0 0 1.25rem;
+        padding: 0 0 0 0;
     }
+
     &__instrukslist {
         margin-top: 1rem;
-        margin-left: 1.25rem;
         display: flex;
         flex-direction: row;
         align-items: center;
     }
+
     &__instrukslistsvg {
-        margin-right: 0.75rem;
+        display: none;
```

Comment:
Isteden for å alltid ha `display: none` og `display: block` når man har desktop-size, kunne man kun hatt `display: none` på mobil  (`max-width` istedenfor `min-width`)